pom.xml

```diff
@@ -381,6 +388,21 @@
             <artifactId>slf4j-api</artifactId>
             <version>1.7.25</version>
         </dependency>
+        <dependency>

```

Comment:
We don't see the protobuf, aws, and google cloud dependencies used in R5 (at least searching through the diff).  If they're being used transitively by another library, could they be included there instead?